.travis.yml

```diff
@@ -1,11 +1,13 @@
+---
 language: ruby
 sudo: false
+cache: bundler
 before_install:
-  - gem install bundler
   - gem update --system
+  - gem install bundler
 dist: trusty
 rvm:
-  - 2.4.0
+  - 2.3.0
```

Comment:
the json gem is in version 2.X in the ruby2.4 version, but one of our dependencies needs 1.X :(
pinning json didn't work out, so I had to downgrade our ruby version